1.20.5/1.20.5-pre3.md

```diff
@@ -0,0 +1,3 @@
+pack breaking | The Data Pack version is now 41
+
+tag | Replaced Entity Type tag `minecraft:punchable_projectiles` with tag `minecraft:redirectable_projectile` for projectiles which can be redirected by player attacks and projectiles
```

Comment:
Can have categories `data` and `tag`. Also don't include `minecraft:`

Reply:
fixed.